keyboards/planck/keymaps/dshields/keymap.c

```diff
@@ -73,12 +72,12 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
      * |      |      |      |      |      |             |      |MsLeft|MsDown| MsUp |MsRght|
      * `-----------------------------------------------------------------------------------'
      */
-    [FUN] = {
-        {KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   RESET,   KC_DEL,  KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10 },
-        {KC_F11,  KC_F12,  KC_F13,  KC_F14,  KC_F15,  _______, _______, _______, KC_WH_L, KC_WH_D, KC_WH_U, KC_WH_R},
-        {BL_TOGG, BL_STEP, _______, _______, _______, DM_STRT, DM_STOP, _______, _______, KC_BTN1, KC_BTN2, KC_BTN3},
-        {_______, _______, _______, _______, _______, _______, _______, _______, KC_MS_L, KC_MS_D, KC_MS_U, KC_MS_R}
-    },
+    [RSE] = LAYOUT_planck_grid(
```

Comment:
I think you accidentally renamed this layer.

Was named `FUN` in the previous revision, and the previous layer here is already named `RSE`.